backend/organization/models/organization_project_published.py

```diff
@@ -0,0 +1,46 @@
+from organization.models.project import Project
+from django.db import models
+from organization.models.organization import Organization
+from django.contrib.auth.models import User
+
+
+class OrgProjectPublished(models.Model):
```

Comment:
Model removed. However, unclear still about how to define the field "org_project_published" currently the foreignkey is changed from OrgProjectPublished to OrganizationFollower, which I also believe is superfluous.

Reply:
We can derive the organization that published a project from the existing data because a project always just has one creator. However your solution could be helpful if we also want to notify followers of collaborating organizations. For now we should not do that because you can just add any org as a collaborator without them needing to confirm but in fact your solution makes it future proof for this type of use-case :+1: